oarepo_ui/theme/assets/semantic-ui/js/oarepo_ui/api/client.js

```diff
@@ -78,43 +81,108 @@ export class DepositApiClient {
  * API Client for deposits.
  */
 export class ApiClient extends DepositApiClient {
-  async _createResponse(axiosRequest) {
+  constructor(createUrl) {
+    super();
+    this.createUrl = createUrl;
+  }
+  _createResponse = async (axiosRequest) => {
     try {
       const response = await axiosRequest();
       const data = response.data || {};
       return data;
     } catch (error) {
-      const errorData = error.response.data;
-      return Promise.reject(errorData);
+      return Promise.reject(error);
     }
-  }
+  };
 
   /**
    * Calls the API to create a new draft.
    *
    * @param {object} draft - Serialized draft
    */
-  async createDraft(url, payload) {
-    return this._createResponse(() => this.axiosWithConfig.post(url, payload));
-  }
+  createDraft = async (draft) => {
+    return this._createResponse(() =>
+      this.axiosWithConfig.post(
+        this.createUrl,
+        // this.createUrl.replace("https://0.0.0.0:5000", ""),
```

Comment:
@mirekys For sure, but we have a small problem, because I don't think we want to do this stripping in code in reality, but if I don't strip it I get that error from https://linear.app/ducesnet/issue/BE-112/issues-with-content-security-policy sometimes. (when I use the links as they come in record, without stripping protocol,host and port). But the issue is kind of random and I don't understand why it is happening.

Reply:
Yep I agree on that, we should fix in BE-112 asap, so that it provides proper links, we shouldn't neet to modify them here in FE